src/Symfony/Component/Form/Tests/Extension/Core/Type/LazyChoiceTypeTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\ChoiceList\Loader\CallbackChoiceLoader;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Test\TypeTestCase as TestCase;
+use Symfony\Component\Form\ChoiceList\View\ChoiceView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Tests children of choice type using a default choice loader.
+ *
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+class LazyChoiceTypeTest extends TestCase
```

Comment:
and btw, AppVeyor runs the right version of ICU, so all PRs are tested properly on our CI.